app/components/views/Offset/index.tsx

```diff
@@ -520,13 +516,14 @@ export const Offset = (props: Props) => {
               })}
             />
           </div>
+
           <div className="disclaimer">
             <GppMaybeOutlined />
             <Text t="caption">
               <Trans id="offset_disclaimer">
-                Be careful not to expose any sensitive personal information. The
-                information you provide will be made a permanent, immutable
-                record on a public blockchain.
+                Be careful not to expose any sensitive personal information.
+                Your message can not be edited and will permanently exist on the
```

Comment:
IMO the whole 'it's public' piece is key to mention:

```suggestion
                Your message can not be edited and will permanently exist on a public
```